src/main/java/menu/ExportJSONMenuHandler.java

```diff
@@ -0,0 +1,63 @@
+package menu;
+
+import dao.factories.DAOFactory;
+import models.*;
+import models.xml.Hospital;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.List;
+
+public class ExportJSONMenuHandler extends MenuHandler {
+    public ExportJSONMenuHandler(DAOFactory factory) {
+        super(factory);
+    }
+
+    @Override
+    public void processMenuOption() {
+        System.out.println("Retrieving information...");
```

Comment:
You should try not to use System.out.println in your projects, because this is unacceptable when working on a project. You can use a logger